Vintageous.sublime-project

```diff
@@ -11,7 +11,8 @@
 	{
 		"ensure_new_line_at_eof_on_save": true,
 		"trim_trailing_white_space_on_save": true,
-		"default_line_ending": "unix"
+		"default_line_ending": "unix",
+		"translate_tabs_to_spaces": false
```

Comment:
If user sets `translate_tabs_to_spaces=true` (linux) some tests fail. I think they fail because of hardcoded tabs `\t` in some assertions on expected text.